src/main/java/com/palpal/dealightbe/config/S3Config.java

```diff
@@ -0,0 +1,39 @@
+package com.palpal.dealightbe.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@Configuration
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class S3Config {
+	@Value("${cloud.aws.credentials.accessKey}")
```

Comment:
```suggestion
public class S3Config {

	@Value("${cloud.aws.credentials.accessKey}")
```
개행 처리 해주시면 될 것 같습니다!